restdocs-openapi-gradle-plugin/src/main/kotlin/com/epages/restdocs/openapi/gradle/OpenApi20Generator.kt

```diff
@@ -105,6 +156,8 @@ internal object OpenApi20Generator {
                         firstModelForPathAndMehtod.request.headers.map {
                             header2Parameter(it)
                         }
+                    ).plus(
+                        requestFieldDescriptor2Parameter(firstModelForPathAndMehtod, modelsWithSamePathAndMethod.map { it.request.requestFields }.reduce{l1, l2 -> l1.plus(l2)})
```

Comment:
I think `.flatten()` is what we were looking for here, instead of the manual reduce